### PR TITLE
First implementation

### PR DESCRIPTION
**Pull request by RamonGilabert**
So here's the website for desktop:

screen shot 2016-01-06 at 2 58 28 pm

And here's for the phone:

screen shot 2016-01-06 at 3 02 15 pm

I've tested most of the cases and sizes. I think it works aight.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <title>iOS + Hyper</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <script type="text/javascript" src="scripts.js"></script>
+
```
Comment (zenangst) [inline public/index.html:8]: Why a newline here?

Comment (RamonGilabert) [inline public/index.html:8]: I like to separate the meta from the title, and imports.

Comment (RamonGilabert) [inline public/index.html:8]: They did it here too though. https://github.com/hyperoslo/hyper-says/blob/master/public/index.html

**Comment by zenangst** [comment]
Have you tested this in Internet Explorer? 🤔

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=0.8, maximum-scale=0.8, user-scalable=no" />
+  </head>
+
+  <body>
+    <section id="header-section">
+      <nav id="navigation">
+        <p></p>
+        <ul>
+          <li id="link-story">Our story</li>
```
Comment (vadymmarkov) [inline public/index.html:20]: Cool story

File: public/scripts.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    footer = document.getElementById("footer"),
+    links = [],
+    sections = [story, openSource, projects, footer],
+    isTouchDevice = 'ontouchstart' in document.documentElement;
+
+  for (var i = 0; i < linksNodeList.length; i++) {
+    links[i] = linksNodeList[i];
+  }
+
+  console.log(isTouchDevice);
```
Comment (vadymmarkov) [inline public/scripts.js]: :scream:

**Comment by zenangst** [comment]
Looks good :sunglasses:

File: public/scripts.js (hunk 3 above)
Comment (zenangst) [inline public/scripts.js]: Oh god!

**Comment by RamonGilabert** [comment]
http://thenextweb.com/microsoft/2016/01/05/web-developers-rejoice-internet-explorer-8-9-and-10-die-on-tuesday/?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+TheNextWeb+%28The+Next+Web+All+Stories%29

@zenangst. If microsoft ain't support it. Ain't going to support it. :smile:

File: public/scripts.js (hunk 3 above)
Comment (RamonGilabert) [inline public/scripts.js]: Oh fuck.

File: public/scripts.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
+document.addEventListener("DOMContentLoaded", function() {
+
+  var linksNodeList = document.getElementById("navigation").getElementsByTagName("li"),
+    linkText = document.getElementById("navigation").getElementsByTagName("p")[0],
+    header = document.getElementById("header-section"),
```
Comment (vadymmarkov) [inline public/scripts.js:5]: You don't like `jQuery`, right?

Comment (RamonGilabert) [inline public/scripts.js:5]: :flushed: I mean, I know JS at the moment.

File: public/index.html (hunk 2 above)
Comment (RamonGilabert) [inline public/index.html:20]: :smile:

File: public/index.html (hunk 1 above)
Comment (zenangst) [inline public/index.html:8]: Nice ;) Just wondering.

**Pull request merged**